backend/Cargo.toml

```diff
@@ -10,7 +10,7 @@ serde = "1.0.188"
 diesel = { version = "= 2.1", features = ["sqlite", "64-column-tables"] }
 diesel_migrations = "2.1"
 chrono = "0.4.26"
-rawtx-rs = "0.1.12"
+rawtx-rs = { git = "https://github.com/andrewtoth/rawtx-rs", branch = "p2a" }
```

Comment:
should use a rawtx-rs release which includes https://github.com/0xB10C/rawtx-rs/pull/26 (once that's available)